GUI/QtQuick/QtQuickAdapter.h

```diff
@@ -207,12 +207,13 @@ class QtQuickAdapter : public QObject, public ApiBusAdapter, public bs::MainLoop
    void showError(const QString&);
    void showFail(const QString&, const QString&);
    void showNotification(QString, QString);
-   void successExport(QString nameExport);
+   void successExport(const QString nameExport);
```

Comment:
If you want it done even more properly, put a const ref here - this will avoid data copying.